sql/catalyst/src/main/scala/org/apache/spark/sql/types/StructType.scala

```diff
@@ -293,8 +294,8 @@ case class StructType(fields: Array[StructField]) extends DataType with Seq[Stru
   override def defaultSize: Int = fields.map(_.dataType.defaultSize).sum
 
   override def simpleString: String = {
-    val fieldTypes = fields.map(field => s"${field.name}:${field.dataType.simpleString}")
-    s"struct<${fieldTypes.mkString(",")}>"
+    val fieldTypes = fields.map(field => s"${field.name}: ${field.dataType.simpleString}")
```

Comment:
TODO maybe this should use a SeqView to avoid eagerly calling toString() on child fields.